src/components/Thanks/ThanksPageDonationOnly.vue

```diff
@@ -0,0 +1,387 @@
+<template>
+	<div>
```

Comment:
I noticed that this new page is missing the `thanks-message` test ID. Do we need that test ID on the new page as well to prevent automated tests from failing?